src/main/java/org/folio/marc4ld/service/ld2marc/mapper/impl/classification/Ld2MarcDdcClassificationMapper.java

```diff
@@ -0,0 +1,85 @@
+package org.folio.marc4ld.service.ld2marc.mapper.impl.classification;
+
+import static org.folio.ld.dictionary.PropertyDictionary.EDITION;
+import static org.folio.ld.dictionary.PropertyDictionary.EDITION_NUMBER;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.CLASSIFICATION;
+import static org.folio.marc4ld.util.Constants.Classification.ABRIDGED;
+import static org.folio.marc4ld.util.Constants.Classification.FULL;
+import static org.folio.marc4ld.util.Constants.Classification.TAG_082;
+import static org.folio.marc4ld.util.Constants.ONE;
+import static org.folio.marc4ld.util.Constants.Q;
+import static org.folio.marc4ld.util.Constants.SPACE;
+import static org.folio.marc4ld.util.Constants.TWO;
+import static org.folio.marc4ld.util.Constants.ZERO;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Predicate;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.MarcFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Ld2MarcDdcClassificationMapper extends AbstractClassificationMapper {
+
+  private static final Set<ResourceTypeDictionary> SUPPORTED_TYPES = Set.of(CLASSIFICATION);
+  private static final String US_LOC = "United States, Library of Congress";
```

Comment:
Hi @askhat-abishev, Instead of checking this label, can we check for the link `http://id.loc.gov/vocabulary/organizations/dlc`?

Reply:
done